staedlma-Berlin.txt

```diff
@@ -0,0 +1,2 @@
+Places to eat
+- FrauMittenmang
```

Comment:
Really like white sausage, any recommendations on places to go?
Also, really like German lagers, where is your favorite place to grab a liter?